public/app/plugins/panel/canvas/module.tsx

```diff
@@ -25,20 +25,29 @@ export const addStandardCanvasEditorOptions = (builder: PanelOptionsEditorBuilde
     defaultValue: true,
   });
 
+  builder.addBooleanSwitch({
+    path: 'quickDataLinkAccess',
+    name: 'Quick data link access',
+    description: 'Enable one click access to data links for elements with a single data link',
```

Comment:
```suggestion
    description: 'Enable one click data links for elements with a single data link',
```

Perhaps this can be re-worded a bit.